sync_block.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
+set -e  # enable exit on error
+
 NETWORK_NAME=$1
 
-# Check if proper argument is provided for network
 if [[ -z $NETWORK_NAME ]]; then
-  echo "Error: Argument not provided. Usage: $0 <network>. Allowed values are 'sepolia' or 'mainnet'."
+  echo "Error: Argument not provided. Usage: $0 <network>."
```

Comment:
Previous version would be more helpful
```suggestion
  echo "Error: Argument not provided. Usage: $0 <network>. Allowed values are 'sepolia' or 'mainnet'."
```